src/Symfony/Component/Messenger/Tests/Transport/RedisExt/RedisExtIntegrationTest.php

```diff
@@ -100,4 +102,76 @@ public function testConnectionSendDelayedMessagesWithSameContent()
         $this->assertEquals($body, $encoded['body']);
         $this->assertEquals($headers, $encoded['headers']);
     }
+
+    public function testConnectionBelowRedeliverTimeout()
+    {
+        // lower redeliver timeout and claim interval
+        $connection = Connection::fromDsn(getenv('MESSENGER_REDIS_DSN'), [], $this->redis);
+
+        $connection->cleanup();
+        $connection->setup();
+
+        $body = '{"message": "Hi"}';
+        $headers = ['type' => DummyMessage::class];
+
+        // Add two messages
+        $connection->add($body, $headers);
+
+        // Read first message with other consumer
+        $this->redis->xreadgroup(
+            'symfony',
+            'other-consumer2',
+            ['messages' => '>'],
+            1
+        );
+
+        // Queue will not have any messages yet
+        $this->assertNull($connection->get());
+
+        sleep(10);
```

Comment:
because of the clockmock we are not really sleeping here 10 seconds. so in redis no 10 seconds will be gone I think we should just set the `claim_interval` to `0` in this test so we are sure the function is called so we only need the above `$this->assertNull($connection->get());` check